L1Trigger/DTTriggerPhase2/plugins/DTTrigPhase2Prod.cc

```diff
@@ -747,6 +1075,15 @@ void DTTrigPhase2Prod::printmP(const string& ss, const metaPrimitive& mP) const
                               << setw(13) << left << mP.chi2 << " r:" << rango(mP);
 }
 
+void DTTrigPhase2Prod::printmP(const metaPrimitive& mP) const {
+  DTSuperLayerId slId(mP.rawId);
+  std::cout << (int)slId << "\t " << setw(2) << left << mP.wi1 << " " << setw(2) << left << mP.wi2 << " " << setw(2)
+            << left << mP.wi3 << " " << setw(2) << left << mP.wi4 << " " << setw(5) << left << mP.tdc1 << " " << setw(5)
+            << left << mP.tdc2 << " " << setw(5) << left << mP.tdc3 << " " << setw(5) << left << mP.tdc4 << " "
+            << setw(10) << right << mP.x << " " << setw(9) << left << mP.tanPhi << " " << setw(5) << left << mP.t0
+            << " " << setw(13) << left << mP.chi2 << " r:" << rango(mP) << std::endl;
+}
+
```

Comment:
Please change `std::cout` to `edm::LogInfo` or `LogDebug`

Reply:
Done